headless-cms/admin/locales/csCZ.ts

```diff
@@ -0,0 +1,84 @@
+export default {
+	"Add article": "Přidat článek",
+	"Add menu": "Přidat menu",
+	"Add page": "Přidat stránku",
+	"Advanced website SEO": "Pokročilá úprava SEO",
+	"Alternative text": "Alternativní text",
+	"Article": "Článek",
+	"Articles": "Články",
+	"Blog": "Blog",
+	"Button": "Tlačítko",
+	"Buttons": "Tlačítka",
+	"Cancel": "Zrušit",
+	"Contact section": "Sekce s kontaktním formulářem",
+	"Content section": "Sekce s obsahem",
+	"Content": "Obsah",
+	"Cover photo": "Titulní obrázek",
+	"Create date": "Datum vytvoření",
+	"Created at": "Vytvořeno",
+	"CTA section": "Sekce s výzvou k akci",
+	"Default": "Výchozí",
+	"Description": "Popis",
+	"E-mail": "E-mail",
+	"Edit article": "Upravit článek",
+	"Edit menu": "Upravit menu",
+	"Edit page": "Upravit stránku",
+	"Error 404": "Chyba 404",
+	"External link": "Externí odkaz",
+	"Features section": "Sekce s funkcemi",
+	"Features": "Funkce",
+	"First name": "Jméno",
+	"Footer copyright": "Copyright v patičce",
+	"Footer": "Patička",
+	"Header": "Hlavička",
+	"Headline": "Nadpis",
+	"Hero section": "Úvodní sekce",
+	"Home page": "Domovská stránka",
+	"Icon": "Ikona",
+	"Id": "Id",
+	"Image": "Obrázek",
+	"Insert link": "Vložit odkaz",
+	"Insert": "Vložit",
+	"Items": "Položky",
+	"Label": "Štítek",
+	"Last name": "Příjmení",
+	"Link": "Odkaz",
+	"Logo": "Logo",
+	"Logos section": "Sekce s logy",
+	"Logos": "Logy",
+	"Menus": "Menu",
+	"Message": "Zpráva",
+	"Messages": "Zprávy",
+	"Name": "Jméno",
+	"Navigation": "Navigace",
+	"OG description": "OG popis",
+	"OG image": "OG obrázek",
+	"OG title": "OG titulek",
+	"Page role": "Role stránky",
+	"Pages": "Stránky",
+	"Perex": "Perex",
+	"Phone number": "Telefonní číslo",
+	"Position": "Pozice",
+	"Preview": "Náhled",
+	"Primary": "Primární",
+	"Publish date": "Datum zveřejnění",
+	"Recommended aspect ratio 19:10 (e.g.: 2400×1260 px).": "Doporučený poměr 19:10 (např. 2400×1260 px).",
+	"Remove link": "Odstranit odkaz",
+	"Role": "Role",
+	"Secondary": "Sekundární",
+	"Select what you want to link.": "Vyberte kam chcete odkazovat.",
+	"SEO articles": "SEO články",
+	"SEO pages": "SEO stránky",
+	"SEO": "SEO",
+	"Settings": "Nastavení",
+	"Short summary of the page. Keep it between 120 and 158 characters.": "Krátký popis stráky. Ideální rozsah je mezi 120 a 158 znaky.",
+	"Show contact message": "Zobrazit zprávu",
+	"Subtitle": "Podtitul",
+	"Testimonial section": "Sekce se zkušentmi",
+	"Testimonials": "Zkušenosti",
```

Comment:
🤨 

https://cs.wikipedia.org/wiki/Testimoni%C3%A1l

![Uploading image.png…]()

Reply:
`Doporučení` is probably the best one.